packages/app/src/pages/gemeente/[code]/vaccinaties.tsx

```diff
@@ -0,0 +1,105 @@
+import { PageInformationBlock } from '~/components/page-information-block';
+import { TileList } from '~/components/tile-list';
+import { GmLayout } from '~/domain/layout/gm-layout';
+import { Layout } from '~/domain/layout/layout';
+import { VaccinePageIntroductionVrGm } from '~/domain/vaccine/vaccine-page-introduction-vr-gm';
+import { useIntl } from '~/intl';
+import { withFeatureNotFoundPage } from '~/lib/features';
+import {
+  createPageArticlesQuery,
+  PageArticlesQueryResult,
+} from '~/queries/create-page-articles-query';
+import { getVaccinePageQuery } from '~/queries/vaccine-page-query';
+import {
+  createGetStaticProps,
+  StaticProps,
+} from '~/static-props/create-get-static-props';
+import {
+  createGetContent,
+  getLastGeneratedDate,
+  selectGmPageMetricData,
+} from '~/static-props/get-data';
+import { VaccinationPageQuery } from '~/types/cms';
+import { replaceVariablesInText } from '~/utils/replace-variables-in-text';
+export { getStaticPaths } from '~/static-paths/vr';
+
+export const getStaticProps = withFeatureNotFoundPage(
+  'gmVaccinationPage',
+  createGetStaticProps(
+    getLastGeneratedDate,
+    selectGmPageMetricData('deceased_rivm', 'difference', 'code'),
```

Comment:
```suggestion
    selectGmPageMetricData('difference', 'code'),
```
AFAICS, the metric `deceased_rivm` isn't used on this page